rng_guesser/number_guessing_game/src/main.rs

```diff
@@ -0,0 +1,59 @@
+use rand::thread_rng;
+use rand::Rng;
+
+use std::io;
+
+fn main() {
+    println!("Program Started...");
+
+    let start_txt = "
+██████╗     ███╗   ██╗     ██████╗              ██████╗ ██╗   ██╗███████╗███████╗███████╗███████╗██████╗ 
+██╔══██╗    ████╗  ██║    ██╔════╝             ██╔════╝ ██║   ██║██╔════╝██╔════╝██╔════╝██╔════╝██╔══██╗
+██████╔╝    ██╔██╗ ██║    ██║  ███╗            ██║  ███╗██║   ██║█████╗  ███████╗███████╗█████╗  ██████╔╝
+██╔══██╗    ██║╚██╗██║    ██║   ██║            ██║   ██║██║   ██║██╔══╝  ╚════██║╚════██║██╔══╝  ██╔══██╗
+██║  ██║    ██║ ╚████║    ╚██████╔╝            ╚██████╔╝╚██████╔╝███████╗███████║███████║███████╗██║  ██║
+╚═╝  ╚═╝    ╚═╝  ╚═══╝     ╚═════╝              ╚═════╝  ╚═════╝ ╚══════╝╚══════╝╚══════╝╚══════╝╚═╝  ╚═╝                                                                                                                                                                                                                                         
+    ";
+
+    println!("{start_txt}");
+    println!("Made by lucasodev - UNLICENSE");
+    println!(" ");
+    println!(" ");
+    
+    guessing();
+}
+
+fn guessing() {
+    let random_number = get_random_number();
```

Comment:
Possible variable naming, would something like `target` or `target_number` be clearer ?

Reply:
Yes, I can fix it when I come back to my pc.